### PR TITLE
add some desc

### PR DESCRIPTION
**Pull request by glkappe**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Most rds do not allow reload permission, so add necessary instructions

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dumpling-overview.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ TiDB 还提供了其他工具，你可以根据需要选择使用：
 ### 需要的权限
 
 - SELECT
-- RELOAD
+- RELOAD（RDS 忽略）
```
Comment (lance6716) [inline dumpling-overview.md:50]: 这个表述太模糊了。可以换成在下面写一个

> **注意：**
>
> 如果上游为 RDS…

**Review by lance6716 — APPROVED** [review]

File: dumpling-overview.md [hunk 2]
```diff
@@ -52,6 +52,10 @@ TiDB 还提供了其他工具，你可以根据需要选择使用：
 - REPLICATION CLIENT
 - PROCESS
 
+> **注意：**
+>
+> 如果上游数据库为 RDS，或者为数据库托管服务，上游用户权限请忽略 `RELOAD`。
```
Comment (shichun-0415) [inline dumpling-overview.md:57]:
```suggestion
> 如果上游数据库为 RDS，或者为数据库托管服务，将不支持上游用户的 `RELOAD` 权限。
```

Comment (lance6716) [reply dumpling-overview.md:57]: 这里要表达的是，RDS 无法为用户增加 RELOAD 权限。但我们已经自动适应了这个场景

Comment (shichun-0415) [reply dumpling-overview.md]: 根据 Frank 的意见，细化了权限使用场景的描述，请再帮忙看看，谢谢：[5d52d8a](https://github.com/pingcap/docs-cn/pull/10884/commits/5d52d8a83acf4d787a78f107c9189e105277f4d3)

File: dumpling-overview.md [hunk 3]
```diff
@@ -46,11 +46,10 @@ TiDB 还提供了其他工具，你可以根据需要选择使用：
 
 ### 需要的权限
 
-- SELECT
-- RELOAD
-- LOCK TABLES
-- REPLICATION CLIENT
-- PROCESS
+- SELECT：导出目标表时需要。
+- RELOAD：使用 consistency flush 时需要。RDS 无法为用户增加该权限，因此，上游为 RDS 或采用托管服务时，可忽略该权限。
```
Comment (Frank945946) [inline dumpling-overview.md:50]: 只有 TiDB 支持该权限，因此，上游为 RDS 或采用托管服务时，可忽略该权限。

**Review by Frank945946 — APPROVED** [review]

File: dumpling-overview.md (hunk 3 above)
Comment (shichun-0415) [inline dumpling-overview.md:50]:
```suggestion
- RELOAD：使用 consistency flush 时需要。注意，只有 TiDB 支持该权限，当上游为 RDS 或采用托管服务时，可忽略该权限。
```

File: dumpling-overview.md [hunk 4]
```diff
@@ -46,11 +46,10 @@ TiDB 还提供了其他工具，你可以根据需要选择使用：
...
 
-- SELECT
-- RELOAD
-- LOCK TABLES
-- REPLICATION CLIENT
-- PROCESS
+- SELECT：导出目标表时需要。
+- RELOAD：使用 consistency flush 时需要。注意，只有 TiDB 支持该权限，当上游为 RDS 或采用托管服务时，可忽略该权限。
+- LOCK TABLES：使用 consistency lock 时需要，需要导出的库表都有该权限。
+- REPLICATION CLIENT：导出 metadata 记录数据快照点时需要，可选。
```
Comment (shichun-0415) [inline dumpling-overview.md:52]:
```suggestion
- REPLICATION CLIENT：导出 metadata 记录数据快照点时需要，可选，如果不需要导出 metadata，可忽略该权限。
```

**Pull request merged**